oasst-shared/oasst_shared/schemas/protocol.py

```diff
@@ -26,7 +26,7 @@ class TaskRequestType(str, enum.Enum):
 class User(BaseModel):
     id: str
     display_name: str
-    auth_method: Literal["discord", "local"]
+    auth_method: Literal["discord", "local", "system"]
```

Comment:
is it ok that this is in this PR?

Reply:
The user created for the importer has `auth_method == "system"`. This user is for example returned by the user_cursor queries .. pydantic otherwise complains when we try to create a FontendUser object for it which is derived from protocol.User ...